lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -259,7 +270,7 @@ let passingCount = 0;
 let failingCount = 0;
 expectations.forEach(expected => {
   console.log(`Checking '${expected.initialUrl}'...`);
-  const results = runLighthouse(expected.initialUrl, configPath);
+  const results = runLighthouse(expected.initialUrl, configPath, cli['save-assets-to']);
```

Comment:
it should be both now that the bug is fixed and we were using the kebab case versions above